docassemble/ALWeaver/data/sources/output_patterns.yml

```diff
@@ -223,6 +229,15 @@ al metadata: |
         "typical role": "${ typical_role }",
         % endif
       })
+# Default country and state
+default country and state: |
+  code: |
+    AL_DEFAULT_COUNTRY = "${ interview.country }"
+  % if hasattr(interview, 'state') and interview.state:
+  ---
+  code: |
+    AL_DEFAULT_STATE = "${ interview.state }"
+  % endif
 # Includes
```

Comment:
See my comment under line 280 in assembly_line.yml regarding "default".

Also, how are these two new fields (country and state) to be used by AL? If they are just convenient handles for author's grab, maybe add some instructions in this block?

Reply:
These fields are used to control the Address fields, so that the correct prompt and correct list of choices is given for State/Province/etc. They shouldn't normally be customized after the interview is generated.